tests/bson/bson-int64-cast_error-001.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+MongoDB\BSON\Int64 casting errors
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+$int64 = new MongoDB\BSON\Int64(2);
+
+echo raises(function () use ($int64) {
+    var_dump((float) $int64);
+}, E_NOTICE | E_WARNING), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+OK: Got %r(E_NOTICE|E_WARNING)%r
+Object of class MongoDB\BSON\Int64 could not be converted to float
```

Comment:
Forgive me if I'm beating a dead horse, but does it still make sense to prohibit an explicit float case knowing that we can so easily return a float from the Int64 arithmetic operations?